Makefile

```diff
@@ -4,8 +4,7 @@ VALIDATOR_IMAGE := "ghcr.io/peggyjv/sommelier-sommelier:main"
 ORCHESTRATOR_IMAGE := "ghcr.io/peggyjv/gravity-bridge-orchestrator:latest"
 
 protos:
-	scripts/build_go_protos.sh
-	scripts/build_api_docs.sh
+	@cargo check --locked
```

Comment:
This might benefit from a comment explaining why `cargo check` builds the protos.